1_prep/src/build_model_config.R

```diff
@@ -28,12 +29,22 @@ build_nldas_model_config <- function(site_ids, nml_list, nldas_csvs, nldas_dates
     )
     return(site_meteo_xwalk)
   })
+
+  site_obs_xwalk <- tibble(
+    site_id = sapply(obs_rds,
```

Comment:
Interesting that you'd need this regex, isn't `site_id` already a field in the `obs_rds`? 

never mind, I see that `obs_rds` is a character vector of branch names and .rds files that have the obs. I get what it is doing now. 

I also agree that splitting out the large obs data into lake-specific files is appropriate here because that way you don't trigger rebuilds when parts of the large file change that aren't relevant to the specific lake being modeled 👍

Reply:
Yep, the `rds` files do not contain the `site_id`, only the name of the file does.